atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/SimpleDbWriteTable.java

```diff
@@ -118,16 +140,30 @@ public void delete(List<Entry<Cell, Long>> entries) {
             Cell cell = entry.getKey();
             args.add(new Object[] {cell.getRowName(), cell.getColumnName(), entry.getValue()});
         }
-        conns.get().updateManyUnregisteredQuery(" /* DELETE_ONE (" + tableName + ") */ "
-                + " DELETE /*+ INDEX(m pk_" + prefixedTableName() + ") */ "
-                + " FROM " + prefixedTableName() + " m "
+
+        conns.get().updateManyUnregisteredQuery(" /* DELETE_ONE (" + tableRef + ") */ "
+                + " DELETE /*+ INDEX(m pk_" + getInternalTableName() + ") */ "
+                + " FROM " + getInternalTableName() + " m "
                 + " WHERE m.row_name = ? "
                 + "  AND m.col_name = ? "
                 + "  AND m.ts = ?",
                 args);
     }
 
+    private String getInternalTableName() {
+        if (!config.type().equals(OracleDdlConfig.TYPE)) {
+            return prefixedTableName();
+        }
+        AgnosticResultSet result = conns.get().selectResultSetUnregisteredQuery(
```

Comment:
see above, but these mappings needs to be cached and ideally pre-fetched at schema load.